### DISCUSSION TITLE
Why we require sigmoid activation over the BCE loss??

### DISCUSSION THREAD

**Discussion by kRiShNa-429407** [mrdbourke/pytorch-deep-learning#946] [Q&A]
Video no : 77
Timestamp  : 10:57

Why we need to use BCEWithLogits over simple BCE??

**Comment by LuluW8071** [comment]
`BCEwithLogitsLoss` is used because it has an in-built sigmoid activation function, allowing you to pass the raw model outputs directly while `BCELoss` expects the output of your model is within the range [0, 1] to represent probabilities.

You can also refer to pytorch documentation if you want more detailed explanation.
